docs/customization.md

```diff
@@ -82,26 +82,30 @@ please refer to Django's guide on [overriding templates][overriding-templates].
 
 ```html
 <!-- hijack/notification.html -->
-<link rel="stylesheet" type="text/css" href="{% static 'hijack/hijack.css' %}" media="screen">
-<div class="djhj" id="djhj">
-  <div class="djhj-notification">
-    <div class="djhj-message">
-      {% blocktrans trimmed with user=request.user %}
-        You are currently working on behalf of <em>{{ user }}</em>.
-      {% endblocktrans %}
+{% load i18n static %}
+
+{% block content %}
```

Comment:
What's the content block for?

Reply:
hahaha, maybe remove it here then. I will drop it from the actual template in a future release.

Reply:
@codingjoe Done ;)

Reply:
I've also noticed that in the repository the css link is correct, but in the online documentation it is not: maybe something wasn't deployed to read the docs? :) I've opened this to correct the css link, and didn't notice it was already corrected.